### ISSUE TITLE
Build system is broken on macos

### ISSUE THREAD

**Issue by haampie** [rhash/RHash#246]
```
./configure --prefix=/something
make
make install
```

on darwin, I'm ending up (a) without pkg config files and (b) with an incorrect symlink:

```
$ ls -l lib/
librhash.1.dylib
librhash.so.1 -> librhash.1.dylib
```

Shouldn't it be `librhash.dylib -> librhash.1.dylib`?

I completely agree with #65, cause it's a major pain to build such a trivial package...

**Comment by rhash** [comment]
Thank you for reporting this bug! It is fixed by commit ee1cf8ce8c1a67ca07fabfba6ba3d6c0909b5fed.

**Comment by rhash** [comment]
The fix is included into [RHash v1.4.5](https://github.com/rhash/RHash/releases/tag/v1.4.5).
